packages/@aws-cdk/cdk/lib/cloudformation/pseudo.ts

```diff
@@ -10,6 +10,49 @@ import { CfnReference } from './cfn-tokens';
  * values can be obtained as properties from an scoped object.
  */
 export class Aws {
+  private constructor() {
+  }
+
+  public static get accountId(): string {
+    return new AwsAccountId(undefined).toString();
+  }
+
+  public static get urlSuffix(): string {
+    return new AwsURLSuffix(undefined).toString();
+  }
+
+  public static get notificationArns(): string[] {
+    return new AwsNotificationARNs(undefined).toList();
+  }
+
+  public static get partition(): string {
+    return new AwsPartition(undefined).toString();
+  }
+
+  public static get region(): string {
+    return new AwsRegion(undefined).toString();
+  }
+
+  public static get stackId(): string {
+    return new AwsStackId(undefined).toString();
```

Comment:
What’s the benefit of having the scope argument in all these ctors?

Reply:
So the consumer is being forced to make a conscious choice about making it scoped or nonscoped. You can't just forget to supply the constructor arg.

Reply:
But what is the meaning of a scoped AwsRegion? Moreover why do we even need those as separate classes? Aren’t then static properties sufficient?

I feel I am missing something :-)

Reply:
It is fundamentally correct to treat region as scoped because it depends on the providing stack, but practically useless because in the only situation in which you would return the value and can usefully use it, it would be the same as the consumer's AWS::Region. 

So yeah, I'll change 😊

Reply:
Was trying to figure out why the comments indicated a scope was needed, this explains why the comment is outdated 

https://github.com/aws/aws-cdk/blob/v2.173.0/packages/aws-cdk-lib/core/lib/cfn-pseudo.ts#L21